web/src/components/review-form.jsx

```diff
@@ -128,13 +129,22 @@ export default class ReviewForm extends React.Component {
           </section>
         )) }
 
+        <section id="confirm-buttons" className="divCenter">
+
+        { this.state.pendingSentences ?
+          <SpinnerButton></SpinnerButton> :
+          <button type="submit">Finish Review</button>
+        }
+
         { this.state.pendingSentences && (
-          <p>
-            <strong>Reviews are being uploaded. This can take several minutes depending on the number of sentences added.
-            Please don't close this website.</strong>
-          </p>
+          <div>
+            <p className="loadingText">Reviews are being uploaded. This can take several minutes depending on the number of sentences added.
+            Please don't close this website.</p>
```

Comment:
very minor comment: can you indent the `Please don't close this website..` by 2 spaces?

Reply:
👍